docs/install-w-machine.md

```diff
@@ -0,0 +1,223 @@
+---
+page_title: Get Started with Docker Swarm
+page_description: Swarm release notes
+page_keywords: docker, swarm, clustering, discovery, release, notes
+---
+
+# Get Started with Docker Swarm
+
+This page introduces you to Docker Swarm by teaching you how to create a swarm
+on your localmachine using Docker Machine and VirtualBox. Once you have a feel
+for creating and interacting with a swarm, you can use what you learned to test
+deployment on a cloud provider or your own network.
+
+Remember, Docker Swarm is currently in BETA, so things are likely to change. We
+don't recommend you use it in production yet.
+
+## Prerequisites
+
+Make sure your local system has VirtualBox installed. If you are using Mac OSX
+or Windows and have installed Docker, you should have VirtualBox already
+installed.
+
+Using the instructions appropriate to your system architecture, [install Docker
+Machine](http://docs.docker.com/machine/#installation).
+
+## Create a Docker Swarm
+
+Docker Machine gets hosts ready to run Docker containers. Each node in your
+Docker Swarm must have access to Docker to pull images and run them in
+containers. Docker Machine manages all this provisioning for your swarm.
+
+Before you create a swarm with `docker-machine`, you create a TLS token. Using
```

Comment:
TLS token -> discovery token

tokens are used for discovery and agent registration, this is not related to TLS or secured communications